app/kubemci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -100,6 +91,37 @@ func (h *HealthCheckSyncer) EnsureHealthCheck(lbName string, ports []ingressbe.S
 	return ensuredHealthChecks, err
 }
 
+func getHealthCheckPath(port ingressbe.ServicePort, clients map[string]kubernetes.Interface) (string, error) {
+	var err error
+	path := ""
+	for cName, c := range clients {
+		probe, prErr := kubeutils.GetProbe(c, port)
+		if prErr != nil {
+			prErr = fmt.Errorf("Error %s when getting probe for port %v", prErr, port)
+			err = multierror.Append(err, prErr)
+			continue
+		}
+		if probe != nil && probe.HTTPGet != nil {
+			if path != "" {
+				// Ensure that all probes have the same health check path
+				if path != probe.HTTPGet.Path {
+					pathErr := fmt.Errorf("probe path '%s' from client '%s' does not match already extracted health check path '%s'", probe.HTTPGet.Path, cName, path)
```

Comment:
change "client %s" to "cluster %s". its the cluster name extracted from kubeconfig.
Also same comment as above about making this error msg more user friendly.  say readiness probe from pod spec